apps/opencs/editor.cpp

```diff
@@ -355,6 +356,8 @@ int CS::Editor::run()
     if (mLocal.empty())
         return 1;
 
+    Misc::Rng::init();
```

Comment:
Are we even using Misc::Rng in the editor? Seems redundant. But if we in fact do use it, it probably should go into the constructor instead.

Reply:
We use RNG in components (in controllers and particles, for example). Since the editor renders scene too, I suppose it is better to init it.